app/pb_kits/playbook/pb_filter/_filter.jsx

```diff
@@ -1,86 +1,222 @@
 /* @flow */
 
-import React from 'react'
-import { Button, Caption, Card, CircleIconButton, Flex, Icon, PbReactPopover, SectionSeparator, Title, TitleCount } from '../'
+import React, { useState } from 'react'
+import {
+  Button,
+  Caption,
+  Card,
+  CircleIconButton,
+  Flex,
+  Icon,
+  List,
+  ListItem,
+  PbReactPopover,
+  SectionSeparator,
+  Title,
+  TitleCount,
+} from '../'
 
 type FilterProps = {
+  background?: Boolean,
   className?: String,
   data?: String,
   id?: String,
   children?: React.Node,
   filters?: Array<Hash>,
-  sortMenu?: Array<Hash>
+  results?: Number,
+  sortMenu?: Array<Hash>,
+  template?: 'default' | 'filter_only' | 'sort_only' | 'single'
```

Comment:
Nice work! Quick question here: are all of these props truly optional? `?` indicates it doesn't have to be passed to the kit in order for the kit to function. If so, then the deconstructed prop list below should set defaults for every optional prop.

Reply:
@thestephenmarshall they are optional for the reason you can have only filters or only sorting , you can show results if you choose and the template is set to default if nothing is given.

Reply:
Makes sense